Specs/Core/GregorianDateSpec.js

```diff
@@ -0,0 +1,300 @@
+import { GregorianDate } from "../../Source/Cesium.js";
+
+describe("Core/GregorianDate", function () {
+  describe("Positive scenarios", function () {
+    it("Constructs any valid date", function () {
+      const validDate = new GregorianDate(2022, 2, 4, 23, 54, 0, 999.9, false);
+      expect(validDate.year).toEqual(2022);
+      expect(validDate.month).toEqual(2);
+      expect(validDate.day).toEqual(4);
+      expect(validDate.hour).toEqual(23);
+      expect(validDate.minute).toEqual(54);
+      expect(validDate.second).toEqual(0);
+      expect(validDate.millisecond).toEqual(999.9);
+      expect(validDate.isLeapSecond).toBeFalsy();
```

Comment:
We want to ensure that `isLeapSecond` is a `boolean` like the documentation is described. However, `toBeFalsy()` doesn't enforce the boolean type: it will coerce non-boolean values into a `false` value (e.g. `null` or 0). Use `toBe(false)` or `toBe(true)` instead.

You can read more about it [here](https://ralzohairi.medium.com/the-difference-between-tobe-true-tobetrue-and-tobetruthy-in-the-jasmine-framework-2dd652ff13a2) if you want more detail about the differences.